src/fides/api/main.py

```diff
@@ -212,29 +209,26 @@ async def setup_server() -> None:
 
     await configure_db(CONFIG.database.sync_database_uri)
 
-    logger.info("Validating SaaS connector templates...")
+    log.info("Validating SaaS connector templates...")
     registry = load_registry(registry_file)
     try:
         db = get_api_session()
         update_saas_configs(registry, db)
     finally:
         db.close()
 
-    logger.info("Running Redis connection test...")
+    log.info("Running Redis connection test...")
 
     try:
         get_cache()
     except (RedisConnectionError, ResponseError) as e:
-        logger.error("Connection to cache failed: %s", Pii(str(e)))
+        log.error("Connection to cache failed: %s", Pii(str(e)))
         return
 
     if not scheduler.running:
         scheduler.start()
 
-    logger.info("Starting scheduled request intake...")
-    initiate_scheduled_request_intake()
```

Comment:
I've snuck this change in here too. This logic isn't in use, so shouldn't be invoked.

Reply:
@seanpreston given your deep knowledge of this stuff, if you think that's something that we should do we should pop open an issue for it

Reply:
I think we also have some good use cases of the apscheduler https://github.com/ethyca/fides/blob/main/src/fides/api/ops/service/privacy_request/request_runner_service.py#L466

This is old but it puts privacy requests that were paused due to webhooks in a failed state if the redis cache times out.  I think there's similar cleanup we could do more privacy request cleanup type tasks with apscheduler.

Reply:
@ThomasLaPiana — I've removed the integration in question in [this PR](https://github.com/ethyca/fides/pull/1570). I saw we used `apscheduler` elsewhere and deliberately left that in.